src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const partner = person.sex === 'f'
+    ? 'husband'
+    : 'wife';
```

Comment:
All right, but there is more advanced way to transform set of one values into another (just FYI): 

```
const SexStatuses = { 'f': 'wife', 'm': 'husband' }

<>.. is my {SexStatuses[person.sex]}</>
```